opennms-doc/guide-install/src/asciidoc/text/minion/introduction.adoc

```diff
@@ -2,16 +2,69 @@
 // Allow GitHub image rendering
 :imagesdir: ../../images
 
-[[gi-install-minion]]
-== OpenNMS Minion
+[[gi-minion]]
+== Minion
 
-_Minion_ gives enterprises the ability to create a globally distributed, scalable monitoring fabric.
+_Minion_ gives the ability to monitor devices and applications which are in isolated networks and hard to reach from a central _{opennms-product-name}_ instance.
+Maintaining a large set of _Firewall_ rules to allow a variety of management protocols is sometimes tidious and hard to set up.
+Especially when communication is done over less reliable networks and _UDP_ based management protocols are used a _Minion_ deployment addresses such issues.
```

Comment:
I'd break this out into two sentences, maybe something like:

Communicating with managed devices over unreliable networks and the use of _UDP_ based management protocols can also be difficult to maintain.
Deploying a _Minion_ can be used to address these issues.